vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -155,6 +162,17 @@ def parse_pod(pod)
         end
       end
 
+      createdby_txt = pod.metadata.annotations.try("kubernetes.io/created-by")
```

Comment:
> @Fryguy @jrafanie it's generated by json unmarshalling.

Can you be more specific?  I'd really like to see how it gets defined.  Is the JSON payload something that a user can post?